apps/src/lab2/views/components/editor/CodeEditor.tsx

```diff
@@ -179,14 +178,6 @@ const CodeEditor: React.FunctionComponent<CodeEditorProps> = ({
     editorEditableCompartment,
   ]);
 
-  // Sets aria-label on the input div once code mirror loads
```

Comment:
I ended up moving this to do it all in one place! So just migrated, not deleted:)